electron/electron-builder.sh

```diff
@@ -150,5 +153,15 @@ TARGET_ARCH="$ARGV_ARCHITECTURE" \
   --c.extraMetadata.packageType="$ARGV_PACKAGE_TYPE"
 popd
 
-rm "$ELECTRON_BUILDER_CONFIG"
+if [ "$ARGV_PACKAGE_TYPE" = "appimage" ]; then
```

Comment:
I was confused by the commit title, this actually reverts the removal of zipping the AppImage.